web/src/lib/components/faces-page/people-card.svelte

```diff
@@ -73,22 +66,16 @@
     {/if}
   </a>
 
-  <div class="absolute right-2 top-2" class:hidden={!showVerticalDots}>
-    <CircleIconButton
-      color="opaque"
+  <div class="absolute top-2 right-2">
+    <ButtonContextMenu
+      buttonClass="icon-white-drop-shadow focus:opacity-100 {showVerticalDots ? 'opacity-100' : 'opacity-0'}"
+      buttonColor="opaque"
+      buttonPadding="2"
+      buttonSize="20"
```

Comment:
I think these props should apply to the button by default, so the `button` prefix looks redundant imo.